.github/workflows/tests.yml

```diff
@@ -149,15 +149,11 @@ jobs:
     services:
       redis:
         image: faasm/redis:0.8.12
-        ports:
```

Comment:
GHA service containers and the job container are connected with a docker-specific network, so there is [no need to publish ports](https://docs.github.com/en/actions/using-containerized-services/about-service-containers#running-jobs-in-a-container):
> You don't need to configure any ports for service containers. By default, all containers that are part of the same Docker network expose all ports to each other, and no ports are exposed outside of the Docker network.